fast-sim-and-analysis/FccFastSimAnalysis.md

```diff
@@ -16,78 +16,277 @@ This tutorial will teach you how to:
 
 ## Part II: Analyse with FCCAnalyses
 
-For this part we start by cloning the FCCAnalyses GitHub repository
+We stay in the same directory where we have produced our samples with ```DelphesPythia8_EDM4HEP```.
+
+For this part we start by making sure we can use FCCAnalyses by checking we can load the library correctly:
 
 ```bash
-git clone https://github.com/HEP-FCC/FCCAnalyses.git
-cd FCCAnalyses
+python
+import ROOT
+ROOT.gSystem.Load("libFCCAnalyses")
+
 ```
 
-This tutorial runs with fcc-edm so we need to checkout the corresponding branch
+if no error message, it means we can load it.
+
+Alternatively, you can check with spack, replacing ```<version>``` by the key4hep version you have set up.
 
 
 ```bash
-git checkout fccedm
+spack find -p -d key4hep-stack@<version> | grep fccanalyses
```

Comment:
or 
`echo $FCCANALYSES`

Reply:
I did not checked it you changed the path of this env var, as I didn't wanted it to point on  a non existing dir as it was the case last week.